### PR TITLE
Chore/overview docs

### PR DESCRIPTION
**Pull request by jmikrut**
## Description

Getting Started docs progress

### CODE REVIEW

File: docs/getting-started/installation.mdx [hunk 1]
```diff
@@ -23,131 +23,151 @@ Payload requires the following software:
...
+     -- page.tsx // your existing home page
+     -- ...etc // whatever else you have
 ```
 
-Here is a list of all properties available to pass through `payload.init`:
-
-#### secret
-
-**Required**. This is a secure string that will be used to authenticate with Payload. It can be random but should be at least 14 characters and be very difficult to guess.
+You can name the `(my-app)` folder anything you want. The name does not matter and will just be used to clarify your directory structure for yourself. Common names might be `(frontend)`, (`app`), or similar.
```
Comment (JarrodMFlesch) [inline docs/getting-started/installation.mdx:80]: (`app`) should be `(app)`

File: docs/getting-started/installation.mdx [hunk 2]
```diff
@@ -23,131 +23,151 @@ Payload requires the following software:
...
 
-Payload uses this secret key to generate secure user tokens (JWT). Behind the scenes, we do not use your secret key to encrypt directly - instead, we first take the secret key and create an encrypted string using the SHA-256 hash function. Then, we reduce the encrypted string to its first 32 characters. This final value is what Payload uses for encryption.
+#### 3 - Add the Payload plugin to your Next.js config
 
-#### config
+Payload has a plugin that it uses to ensure Next.js compatibility with some of the packages Payload relies on, like `mongodb` or `drizzle-kit`.
 
-Allows you to pass your config directly to the onInit function. The config passed here should match the payload.config file.
+You need to import the `withPayload` plugin in your `next.config` file and wrap your config with it.
```
Comment (JarrodMFlesch) [inline docs/getting-started/installation.mdx:86]: `next.config` should be `next.config.js` for total clarity

File: docs/getting-started/installation.mdx [hunk 3]
```diff
@@ -23,131 +23,151 @@ Payload requires the following software:
...
 
-Here is a list of all properties available to pass through `payload.init`:
-
-#### secret
-
-**Required**. This is a secure string that will be used to authenticate with Payload. It can be random but should be at least 14 characters and be very difficult to guess.
+You can name the `(my-app)` folder anything you want. The name does not matter and will just be used to clarify your directory structure for yourself. Common names might be `(frontend)`, (`app`), or similar.
 
-Payload uses this secret key to generate secure user tokens (JWT). Behind the scenes, we do not use your secret key to encrypt directly - instead, we first take the secret key and create an encrypted string using the SHA-256 hash function. Then, we reduce the encrypted string to its first 32 characters. This final value is what Payload uses for encryption.
+#### 3 - Add the Payload plugin to your Next.js config
```
Comment (JarrodMFlesch) [inline docs/getting-started/installation.mdx:82]: This looks like the only title that is capital case

File: docs/getting-started/what-is-payload.mdx [hunk 4]
```diff
@@ -11,56 +11,107 @@ keywords: documentation, getting started, guide, Content Management System, cms,
...
+It's fully open source with an MIT license and you can self-host anywhere that you can run a Node app. You can also deploy serverless to hosts like Vercel, right inside your existing Next.js app folder.
 
-## What does "headless" mean?
+Even in spite of how much you get out of the box, you still have full control over every aspect of your app - be it database, admin UI, or anything else. Every part of Payload has been designed to be extensible and customizable with modern TypeScript / React. And you'll fully understand the code that you write.
 
-A headless CMS is a system that sticks to what it's good at—managing content. It concentrates solely on granting administrators an effective way to author and maintain content, but doesn't control how and where that content is used.
+In Payload, there are no "click ops" - as in clicking around in an admin panel to define your schema. In Payload, everything is done the right way—code-first and version controlled like a proper backend. But once developers define how Payload should work, non-technical users can independently make use of its admin panel to manage whatever they need to do without having to know code whatsoever.
```
Comment (JarrodMFlesch) [inline docs/getting-started/what-is-payload.mdx:31]: "make use of its admin panel to manage whatever they need to do"

maybe should be:

"make use of its admin panel to manage whatever they need to"?

File: docs/getting-started/installation.mdx (hunk 1 above)
Comment (jacobsfletch) [reply docs/getting-started/installation.mdx:80]: Could link out to `../admin/overview` from here. That page is a continuation of this information.

Comment (jmikrut) [reply docs/getting-started/installation.mdx:80]: done and done

**Pull request merged**